git/git.go

```diff
@@ -21,7 +21,7 @@ func GitLog(source string, logOpts string) (<-chan *gitdiff.File, error) {
 		args = append(args, strings.Split(logOpts, " ")...)
 		cmd = exec.Command("git", args...)
 	} else {
-		cmd = exec.Command("git", "-C", sourceClean, "log", "-p", "-U0", "--full-history", "--simplify-merges", "--show-pulls", "--all")
+		cmd = exec.Command("git", "-C", sourceClean, "log", "-p", "-U0", "--full-history", "--show-pulls", "--all")
```

Comment:
I'm confused. `--show-pulls` is still here. Am I missing something?

Reply:
🤦🏻 and I committed the wrong thing. Let me fix and release 8.0.2

Reply:
@carlocab https://github.com/zricethezav/gitleaks/commit/a37822dffd90ad27fe4aa36c392ea3cc4a6330d8 released with https://github.com/zricethezav/gitleaks/releases/tag/v8.0.2

Thanks for your patience 😄